content/pytorch/concepts/tensors/terms/zeros/zeros.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'ZEROS'
+Description: 'Initializing a new tensor with zeros.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+  - 'AI'
+Tags:
+  - 'AI'
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/machine-learning'
+---
+
+
+The `.zeros()` method returns a [Tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) of a specified shape with zeros. Optionally, the output tensor, datatype, layout, device, and whether autograd records operations can be specified.
+
+## Syntax
+
+```py
+import torch
+
+torch.zeros(shape)
+
+# with defaults explicitly stated
+torch.zeros(shape, out=None, dtype=torch.float32, layout=torch.strided, device=torch.device('cpu'), requires_grad=False)
```

Comment:
```suggestion
torch.zeros(size, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False)
```

Reply:
Reefrence: https://pytorch.org/docs/stable/generated/torch.zeros.html

Reply:
I see. I saw that in the description it said that if the dtype is written as None, the global default will be assigned, which, if not assigned, is torch.float32. Should we still leave it as none?